packages/xrpl/test/testUtils.ts

```diff
@@ -79,6 +80,41 @@ export async function assertRejects(
   }
 }
 
+const lastSocketKeyMap: { [port: string]: number } = {}
+const socketMap: {
```

Comment:
Why is this done with global variables?

Reply:
It could be refactored, would probably make more sense for all of this to be in a class.